src/libinjector/win_injector.c

```diff
@@ -564,10 +564,11 @@ static bool injector_set_hijacked(injector_t injector, drakvuf_trap_info_t* info
 
 static void fill_created_process_info(injector_t injector, drakvuf_trap_info_t* info)
 {
-    ACCESS_CONTEXT(ctx);
-    ctx.translate_mechanism = VMI_TM_PROCESS_DTB;
-    ctx.dtb = info->regs->cr3;
-    ctx.addr = injector->process_info;
+    ACCESS_CONTEXT(ctx,
+        .translate_mechanism = VMI_TM_PROCESS_DTB,
+        .dtb = info->regs->cr3,
+        .addr = injector->process_info,
+    );
```

Comment:
Not entirely sure what's the point of this refactor?

Reply:
Unification.

Reply:
It's fine as-is, I don't think this is necessary